src/main/java/Cubone.java

```diff
@@ -0,0 +1,89 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+public class Cubone {
+    public static void main(String[] args) {
+        String logo =   
+            "   ______      __                       \n"+
+            "  / ____/_  __/ /_  ____  ____  ___     \n"+
+            " / /   / / / / __ \\/ __ \\/ __ \\/ _ \\\n"+
+            "/ /___/ /_/ / /_/ / /_/ / / / /  __/    \n"+
+            "\\____/\\__,_/_.___/\\____/_/ /_/\\___/ \n";
+        String chat_prefix = "\n(Cubone) ";
```

Comment:
I think we are supposed to use [camel case](https://en.wikipedia.org/wiki/Camel_case) for java and this course